smart_open/gcs.py

```diff
@@ -420,14 +420,20 @@ def __init__(
 
         self._session = google.auth.transport.requests.AuthorizedSession(client._credentials)
 
+        kwargs = {}
+
         if blob_properties:
+            if 'predefined_acl' in blob_properties:
+                kwargs['predefined_acl'] = blob_properties.pop('predefined_acl')
             for k, v in blob_properties.items():
                 setattr(self._blob, k, v)
 
+        if 'predefined_acl' in kwargs and google.cloud.storage.__version__.split('.') < ['2', '6', '0']:
+            raise NotImplementedError('GCS "predefined_acl" requires google-cloud-storage>=2.6.0')
```

Comment:
@mpenkov would this be the preferred way to handle the new feature?

Reply:
not very performant to check this every time. will make a flag on global scope. is `NotImplementedError` ok like this though?